test/config/configurations/full.json

```diff
@@ -27,6 +27,7 @@
             "127.0.0.9",
             "127.0.0.2:5702"
         ],
+        "redoOperation": true,
```

Comment:
I guess we need to add this to `config-schema.json` also

Reply:
Good point. Added the definition in a23763524422f15b6f73f37af1ca99399fe3c126